deployment/helm/node-feature-discovery/templates/topologyupdater.yaml

```diff
@@ -99,6 +104,9 @@ spec:
           {{- else }}
           path: /var/lib/kubelet/pod-resources/kubelet.sock
           {{- end }}
+      - name: kubelet-state-files
```

Comment:
We probably need/want to enclose this inside `{{ if }}` block too. Kubernetes might not be happy if `path` is an empty string

Reply:
Done, Thanks